src/makeInfinityAdder.js

```diff
@@ -6,7 +6,101 @@
  */
 
 function makeInfinityAdder() {
```

Comment:
In this task the easiest way to create InfiniteAdder would be to create function factory and return the function which can return itself with use of Recursion https://developer.mozilla.org/en-US/docs/Glossary/Recursion.

Try to modify your code using this example as a reference:
```js
function recurse() {
    if(condition) {
        // stop calling itself
        //...
    } else {
        recurse();
    }
}
```

Reply:
I try with code like below but it is not working

 ` let result = 0;

  function sum(a) {
    if (a === undefined) {
      return result;
    } else {
      result += a;
      sum();
    }
  }

  return sum;`